iotlabcli/integration/test_cli_tools_help.py

```diff
@@ -0,0 +1,36 @@
+import pytest
```

Comment:
I know. But the tox.ini verifies that the script can be called and that's it. The internal behavior : checking that the options are taken into account correctly is done in the parser tests.
We don't need to write python code that tests things already tested somewhere else.